static/src/javascripts-legacy/bootstraps/commercial.js

```diff
@@ -132,9 +132,8 @@ define([
             robust.context([
                 ['ga-user-timing-commercial-end', function () {
                     ga.trackPerformance('Javascript Load', 'commercialEnd', 'Commercial end parse time');
-                }],
+                }]
             ]);
-            performanceLogging.reportTrackingData();
```

Comment:
because it assumes the promises complete, and timing is done. which isn't true for `defer` based modules